modules/vba_documents/lib/vba_documents/upload_validator.rb

```diff
@@ -84,6 +86,7 @@ def perfect_metadata(model, parts, timestamp)
       end
       metadata['businessLine'] = VALID_LOB[metadata['businessLine'].to_s.upcase] if metadata.key? 'businessLine'
       metadata['businessLine'] = AppealsApi::LineOfBusiness.new(model).value if model.appeals_consumer?
+      metadata['zipCode'] = metadata['zipCode'].to_s if metadata['zipCode'].is_a? Numeric
```

Comment:
I was a little unsure about whether to include this line - it coerces a numeric zip code to a string wherever `perfect_metadata` is called. This includes [here in `download_and_process`](https://github.com/department-of-veterans-affairs/vets-api/blob/6afa56f08e6027644a59bc0b34cbf4413aa38dd0/modules/vba_documents/app/sidekiq/vba_documents/upload_processor.rb#L94), just before we submit to central mail.

Regardless of whether we keep this, the validation process for metadata will still raise a DOC102 when a numeric zip code is supplied (there's a test case for that). I only included this because it seemed like it fit in next to the other corrections in this method (we may fill in a correct `businessLine` for the consumer here, for example). That said, I can see how this could be seen as allowing the user to skirt the schema rules for the metadata if `validate_metadata` is never called for some reason.

Reply:
I decided to just get rid of this - it seems like it would never be used

Reply:
I agree that was the right call!